tests/browseruse/test_runner.py

```
@@ -32,18 +35,34 @@
+                print(f"✅ Test {test_name} PASSED")
             else:
                 failed += 1
+                failed_tests.append(test_name)
+                print(f"❌ Test {test_name} FAILED")
         except Exception as e:
             print(f"❌ Test {test_name} failed with exception: {e}")
+            import traceback
+            traceback.print_exc()
             failed += 1
+            failed_tests.append(test_name)
         
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)